commercial measures/SWSV016-01 Weather Sealing/SWSV016-01 Weather Sealing_Ex/cases/cDXOH.csv

```diff
@@ -0,0 +1,4 @@
+skip,case_name,sizing_case,:heat_sizing_factor,:cool_sizing_factor,:hvac_config,:main_hvac_type,:main_atu_reheat_coil_type,:main_atu_reheat_control,:main_fan_rise,:main_cool_coil_type,:main_heat_coil_type,:main_dx_type,:main_sat_reset_type,:main_sat_temp_max,:main_design_peak_load,:main_oa_econ_type,:main_cool_coil_cop,:main_dx_comp_speed,:main_cool_flow_min_frac,:main_unitary_no_load_flow_method,:main_fan_speed,:main_fan_motor_eff,:main_heat_coil_cop,:main_heat_coil_eff,:main_heat_flow_frac,:chw_chiller_cop,:chw_chiller_model,:chw_pump_type,:hw_boiler_eff,:infil_area
+,NG-BldgShell-Weatherize-0.3-infil,,1.25,1.15,,SZ-CAV,NONE,nil,3.877['in H2O'],DX-AIR,COMBUSTION,PNNL,FLOATING,95['F'],NONCOINCIDENT,FIXED-DRY-BULB,3.65,CONSTANT,,,CONSTANT,0.87,,0.8,1,,,,,0.2995405['CFM/ft2']
```

Comment:
In cDXOH.csv, the :main_heat_coil_type column does not match the reference file from commercial measures/SWXX000-00 TRC System Types/SWXX000-00 TRC System Types_Ex/cases. It looks like a copy of cDXGF instead of cDXOH. There might be other columns, so better to grab the reference cDXOH file from that folder again.